Orange/widgets/data/owmergedata.py

```diff
@@ -256,7 +256,8 @@ class Inputs:
     class Outputs:
         data = Output("Data",
                       Orange.data.Table,
-                      replaces=["Merged Data A+B", "Merged Data B+A", "Merged Data"])
+                      replaces=["Merged Data A+B", "Merged Data B+A", "Merged Data"],
+                      dynamic=False)
```

Comment:
This one I'm unsure of. Could I not merge two corpora? I do this often (append metadata to a corpus).

Reply:
From what I can gather this will always create a plain Table
https://github.com/biolab/orange3/blob/3.38.1/Orange/widgets/data/owmergedata.py#L630-L632
```python
        table = Orange.data.Table.from_numpy(domain, X, Y, metas)
        table.name = getattr(self.data, 'name', '')
        table.attributes = getattr(self.data, 'attributes', {})
```